mpc/setup.py

```diff
@@ -33,6 +33,11 @@
         "ecdsa==0.13.3",
         "click==7.0",
         "requests==2.22.0",
+        # flask constraints on dependencies are to loose. Constrain the version
```

Comment:
- `are to loose` -> `are too loose`?
- `Constrain the version them here` -> I think this needs rephrasing, maybe `constrain the versions here to avoid [...]`?

Reply:
Ah yes.  Thanks for that.
Updated.